robot/Cumulus/resources/ManageHouseHoldPageObject.py

```diff
@@ -43,7 +46,12 @@ def add_contact(self, option, value):
          Supported options are (New/Existing)
         """
         self.npsp.choose_frame("Manage Household")
-        self.npsp.search_field_by_value("Find a Contact or add a new Contact to the Household", value)
+        xpath = npsp_lex_locators['manage_hh_page']['lookup'].format("Find a Contact or add a new Contact to the Household.")
+        field = self.selenium.get_webelement(xpath)
+        self.selenium.clear_element_text(field)
+        field.send_keys(value)
+        time.sleep(3)
```

Comment:
Is there a reason why you are using `time.sleep` instead of ``self.builtin.sleep(3)`? I think this provides logging functionality.

Reply:
no specific reason. I didn't realize there was a builtin sleep keyword when I first using this 2 years back and had been using that. But good point, I will update in the other branch that I have for all lint fixes